Yoast/javascript
# Make SvgIcon able to accept alternative icon sets

## Summary

Makes `SvgIcon` able to accept alternative icon sets by passing the icons object as a prop. This allows to re-use the component in other context where there may be the need to use different icons. See for example https://github.com/Yoast/my-yoast/issues/1576#issuecomment-483613494

## Relevant technical choices:
- exposing the icon set as an `iconSet` prop allows to manipulate the icons object "in place" where the `SvgIcon` is used
  - add icons to the set
  - remove icons from the set
  - pass a completely different icon set
- note: the `SvgIcon` prop `iconSet` will now contain the whole icons object, not sure this is a problem in terms of performance

## Test instructions
- yarn start the apps components and go to localhost:3333 > SVG Icons tab
- see the examples in the page
- see usage examples in `apps/components/SvgIconsWrapper.js`
- check the icons render correctly where used in the other components
- check the icons still render correctly in the Yoast SEO plugin

## Impact check
* This PR affects the following parts of the plugin, which may require extra testing:
  * SVG icons

## UI changes

Fixes #229
